io.github.rdybka.vht.yml

```diff
@@ -0,0 +1,21 @@
+id: io.github.rdybka.vht
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: vht
+finish-args:
+  - --filesystem=xdg-config
+  - --filesystem=xdg-data
```

Comment:
no it shouldn't need these two.

Reply:
"just change where u get ur config path from, dude" - would have worked first time :)
Like I said, I'm new to this and sometimes assume things have to be certain way when they don't.